### PR TITLE
Add kafka page

### PR DESCRIPTION
**Pull request by mohin7**

### CODE REVIEW

File: data/products/kubedb/databases/kafka.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  ],
+  "example": {
+    "apiVersion": "kubedb.com/v1alpha2",
+    "kind": "Kafka",
+    "metadata": {
+      "name": "kafka-quickstart",
+      "namespace": "demo"
+    },
+    "spec": {
+      "version": "3.3.0",
```
Comment (raihankhan) [inline data/products/kubedb/databases/kafka.json]: Use version "3.4.0"

File: content/overview/kubedb/databases/kafka.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Run and Manage Kafka Database on Kubernetes - KubeDB
```
Comment (raihankhan) [inline content/overview/kubedb/databases/kafka.md]: Change the title to -
`Run and Manage Apache Kafka on Kubernetes - KubeDB`
As kafka is not actually a database.

**Review by raihankhan — CHANGES_REQUESTED** [review]

File: data/products/kubedb/databases/kafka.json (hunk 1 above)
Comment (mohin7) [reply data/products/kubedb/databases/kafka.json]: Resolved

File: content/overview/kubedb/databases/kafka.md (hunk 2 above)
Comment (mohin7) [reply content/overview/kubedb/databases/kafka.md]: Resolved

File: data/products/kubedb/databases/kafka.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  ],
+  "example": {
+    "apiVersion": "kubedb.com/v1alpha2",
+    "kind": "Kafka",
+    "metadata": {
+      "name": "kafka-quickstart",
+      "namespace": "demo"
+    },
+    "spec": {
+      "version": "3.4.0",
```
Comment (raihankhan) [inline data/products/kubedb/databases/kafka.json:63]: Add  `"replicas": "3",`

**Review by raihankhan — CHANGES_REQUESTED** [review]

File: data/products/kubedb/databases/kafka.json (hunk 3 above)
Comment (mohin7) [reply data/products/kubedb/databases/kafka.json:63]: resolved in static-assets file

**Review by raihankhan — APPROVED** [review]
LGTM!

**Pull request merged**